sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpSonarWayProfile.java

```diff
@@ -19,16 +19,15 @@
  */
 package org.sonar.plugins.csharp;
 
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
 import org.sonar.api.profiles.ProfileDefinition;
 import org.sonar.api.profiles.RulesProfile;
 import org.sonar.api.profiles.XMLProfileParser;
 import org.sonar.api.utils.ValidationMessages;
 
-import java.io.InputStreamReader;
-import java.nio.charset.StandardCharsets;
-
 public class CSharpSonarWayProfile extends ProfileDefinition {
-
+  private static final String PROFILE_XML_PATH = "/org/sonar/plugins/csharp/profile.xml";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1075)